fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -0,0 +1,31 @@
+require 'pry'
+module Fastlane
+  class MarkdownDocsGenerator
+    attr_accessor :categories
+
+    def initialize
+      require 'fastlane'
+      require 'fastlane/documentation/actions_list'
+      Fastlane.load_actions
+
+      self.work
+    end
+
+    def work
+      self.categories = {}
+      ActionsList.all_actions do |action|
+        self.categories['Misc'] ||= []
+        self.categories['Misc'] << action
+      end
+    end
+
+    def save_to_ssd!(target_path: "docs/ActionsAuto.md")
```

Comment:
This method name is a little strang :smile: `generate!` would probably be easier to understand and guess 👍